code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -592,6 +592,29 @@
 	cover.name = name
 	cover.desc = desc
 
+/obj/machinery/porta_turret/centcomm_shuttle

```

Comment:
I will try and do that, AnthurK helped me out in coderbus with the code for thus due to me being bad at code but ill try this myself anyway.